web/src/message_lists.ts

```diff
@@ -17,51 +16,7 @@ export type SelectIdOpts = {
     mark_read?: boolean;
     force_rerender?: boolean;
     from_scroll?: boolean;
```

Comment:
This type probably is in the wrong file now too, right?

Reply:
Ah yeah, it's only used in `message_list`. Moved, thanks!